spi/build.gradle

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import com.github.jengelman.gradle.plugins.shadow.ShadowBasePlugin
+import org.opensearch.gradle.test.RestIntegTestTask
+
+plugins {
+    id 'com.github.johnrengelman.shadow'
+    id 'jacoco'
+    id 'maven-publish'
+    id 'signing'
+}
+
+apply plugin: 'opensearch.java'
+apply plugin: 'opensearch.testclusters'
+apply plugin: 'opensearch.java-rest-test'
```

Comment:
Nit: As there is no integ test for spi, those two plugins may not needed.

Reply:
Makes sense, will remove.